app/jobs/views.py

```diff
@@ -0,0 +1,22 @@
+# Third-Party imports
+from django.template.response import TemplateResponse
+
+# gitcoin-web imports
+from .models import Job
+
+
+def list_jobs(request):
```

Comment:
This would be changed and just contain template response, since the views just calls the relevant API to fetch data.

@owocki